Gems/SceneProcessing/Code/Source/Generation/Components/MeshOptimizer/MeshOptimizerComponent.cpp

```diff
@@ -167,6 +184,7 @@ namespace AZ::SceneGenerationComponents
             ) * m_positionTolerance;
         }
 
+        AZStd::unordered_map<AZ::Vector3, AZ::u32> m_map;
```

Comment:
can you rename this to position to index map and add some comment to the usage here?

Reply:
That's present in the class's docstring:

```
    // @brief A class to map from a mesh's vertex index to it's welded vertex index
    //
    // When the mesh optimizer runs, it welds nearby vertices (if there are no blendshapes). This class provides a
    // constant time lookup to map from an unwelded vertex index to the welded one.
    // The welding works by rounding the vertex's position to the given position tolerance, then uses that rounded
    // Vector3 as a key into a unordered_map.
```

Do you think we need more than that?